docs/apppromote.yaml

```
@@ -0,0 +1,52 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: apppromote
```

This might be easier to read?

```suggestion
  name: app-promote
```

Along with renaming the file?

But, I think that it should also be really about `service-promote` (same for the other bits) given that the tool talks about promotion of services.

---

I change it.  I also change the file names, too.